### ISSUE TITLE
Add translation for `skipLinkText` used by the Navigation component

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#3151]
**Is your feature request related to a problem? Please describe.**
We have implemented the Navigation component on the live radio pages, however we have not add the translation `skipLinkText` for the 5 services. This is causing a propType warning and is also an a11y bug due to the link not containing any text to explain its purpose. 

![image](https://user-images.githubusercontent.com/7791726/63038643-815b9200-beb9-11e9-9831-151f2e94f471.png)

**Describe the solution you'd like**
For the 5 services add a translation for `skipLinkText`, this can likely be copied over from the production pages but this should be confirm with Product. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: When running `npm run dev` ensure no propType warnings are present on the 5 services with liveradio. 

**Additional context**
We should use a screen reader to check the skip link is read out as expected when the translation is supplied.

**Comment by pjlee11** [comment]
This will be fixed by #3149
